doc/api_assets/style.css

```
@@ -953,6 +953,8 @@ kbd {
   background-repeat: no-repeat;
   width: 142px;
   height: 20px;
+  display: block;
+  cursor: pointer;
 }
```

Alternative to https://github.com/nodejs/node/pull/43506#discussion_r904815683: how about resetting margin on floating switches instead?
```suggestion
  margin: 0;
}
```

---

How would that help?

---

This will avoid the switch jumping a bit due to default margin for `<input>` when resizing the window.

Originally (before https://github.com/nodejs/node/pull/43495#discussion_r901356877) resetting top/left margins for "fixed" layout was necessary because it was aligned to the left and therefore visually aligned with text.
In current version, top/right margins for "fixed" and "regular" versions should match.